### PR TITLE
At7/ld widget

### PR DESCRIPTION
**Pull request by at7**
This functionality has been requested by the web team. It is needed for the LD widget and reduces the number of required rest calls. With mu updates to ld_vcf.c it should be fast enough.

### CODE REVIEW

File: root/documentation/ld.conf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <endpoints>
   <ld_id_get>
     description=Computes and returns LD values between the given variant and all other variants in a window centered around the given variant. The window size is set to 500 kb.    
-    endpoint=ld/:species/:id/:population_name/:d_prime/:r2/:window_size
+    endpoint=ld/:species/:id/:population_name/:d_prime/:r2/:window_size/:attribs
```
Comment (nerdstrike) [inline root/documentation/ld.conf]: I believe this isn't correct. :variable implies part of the URL, not simply ?options you can slap on the end

Comment (at7) [inline root/documentation/ld.conf]: I see. Only the required parameters are listed here? then I also need to remove :d_prime/:r2/:window_size. Is that correct?

Comment (nerdstrike) [inline root/documentation/ld.conf]: That's more like it. endpoint is a URL spec, the options are enumerated below

**Comment by magaliruffier** [comment]
I think this is not good for merging

**Pull request merged**